scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,63 +2,31 @@
 
 # fail if some commands fails
 set -e
+# Hide command that deals with secrets
+set +x
+if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
+    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
+fi
+if [[ -z $AMQP_URI ]]; then
```

Comment:
Shouldn't this be set as `else` instead? Something like:
```sh
if [[ -f $ODO_RABBITMQ_AMQP_URL ]]; then
    export AMQP_URI=$(cat $ODO_RABBITMQ_AMQP_URL)
else
    echo "Please set AMQP_URI env with amqp uri or provide path of file containing it as ODO_RABBITMQ_AMQP_URL env"
    exit 1
fi
```

Reply:
Hmm makes sense